ui/packages/ui/src/Pages/Viewer/Components/Miscellaneous/CharacterActionsBarChart/BarChart.tsx

```diff
@@ -0,0 +1,114 @@
+import { SourceStat, SourceStats } from "@gcsim/types";
+import { LegendOrdinal } from "@visx/legend";
+import { scaleOrdinal } from "@visx/scale";
+import { memo, useMemo } from "react";
+import { DataColors, FloatStatTooltipContent, HorizontalBarStack, NoData } from "../../Util";
+
+type Props = {
+  width: number;
+  height: number;
+  names?: string[];
+  actions?: SourceStats[];
+  actionNames?: string[] | null;
+}
+
+const margin = { top: 0, left: 300, right: 20, bottom: 40 };
```

Comment:
Should set the left margin similar to the Character DPS bar graphs (I need to write something that is more dynmaic and based on max number of characters in label or something).